main.py

```diff
@@ -526,6 +526,21 @@ def load_evcs(self, event):
                         evc.deploy()
                     self.sched.add(evc)
 
+    @listen_to('kytos/flow_manager.flow.error')
+    def handle_flow_mod_error(self, event):
+        """Handle flow mod errors related to an EVC."""
+        flow = event.content['flow']
+        command = event.content.get('error_command')
+        if command is None:
+            return
+        log.info(f'Flow error: {flow.cookie}')
```

Comment:
Should this really be a `log.info`?
If yes, maybe it would be more useful if we included additional information to it, like `error_command` and more `flow` info.

Reply:
Removed